app/assets/javascripts/common/reports.js

```diff
@@ -1106,16 +1103,15 @@ function baseLineGraphConfig() {
               family: "Roboto",
             },
             padding: 6,
+            showLabelBackdrop: true,
           },
           beginAtZero: true,
           min: 0,
         },
         y: {
           stacked: false,
-          display: true,
-          grid: {
-            display: true,
-            drawBorder: false,
```

Comment:
default grid - display = true 
removed as not required 
https://www.chartjs.org/docs/latest/axes/styling.html#grid-line-configuration